packages/backend/src/webhook/service.ts

```diff
@@ -161,12 +161,18 @@ async function processNextWebhook(
         })
       }
 
-      const settings = await deps_.tenantSettingService.get({
-        tenantId: webhook.recipientTenantId
-      })
-      const formattedSettings = formatSettings(settings)
+      if (webhook.metadata?.sendToPosService) {
+        await sendWebhook(deps, webhook, {
+          webhookUrl: `${deps.config.posServiceUrl}/webhook`
```

Comment:
maybe it would be good to have a separate posWebhookServiceUrl (since it can be hosted on a separate service). If you agree, we can turn it into a separate issue

Reply:
Captured in https://github.com/interledger/rafiki/issues/3606 / [RAF-1145](https://linear.app/interledger/issue/RAF-1145/rafiki-backend-add-pos-webhook-service-url)